libraries/AP_Scripting/drivers/BattMon_ANX.lua

```diff
@@ -0,0 +1,264 @@
+--[[ 
+ device driver for ANX CAN battery monitor
+--]]
+
+local MAV_SEVERITY = {EMERGENCY=0, ALERT=1, CRITICAL=2, ERROR=3, WARNING=4, NOTICE=5, INFO=6, DEBUG=7}
+
+local PARAM_TABLE_KEY = 45
+local PARAM_TABLE_PREFIX = "BATT_ANX_"
+
+-- bind a parameter to a variable given
+function bind_param(name)
+    local p = Parameter()
+    assert(p:init(name), string.format('could not find %s parameter', name))
+    return p
+end
+
+-- add a parameter and bind it to a variable
+function bind_add_param(name, idx, default_value)
+    assert(param:add_param(PARAM_TABLE_KEY, idx, name, default_value), string.format('could not add param %s', name))
+    return bind_param(PARAM_TABLE_PREFIX .. name)
```

Comment:
Can use the param constructor.
```suggestion
    return Parameter(PARAM_TABLE_PREFIX .. name)
```

Reply:
done